clusterloader2/README.md

```diff
@@ -1,21 +1,77 @@
 # ClusterLoader
 
-### Running ClusterLoader
+## Running ClusterLoader
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.json
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml
 ```
 Flags kubeconfig and testconfig are necessary.
 
-#### Flags
+### Flags
 
+#### Required
+
+These flags are required for any test to be run.
  - kubeconfig - path to the kubeconfig file.
- - testconfig - path to the test config file.
+ - testconfig - path to the test config file. This flag can be used multiple times
+if more than one test should be run.
 
-### Vendor
+#### Optional
 
-Vendor is created using [govendor].
+ - nodes - number of nodes in the cluster.
+If not provided, test will assign the number of schedulable cluster nodes.
+ - report-dir - path to directory, where summaries files should be stored.
+If not specified, summaries are printed to standard log.
+
+## Tests
+
+### Test definition
+
+Test definition is an instantiation of this [api] (in json or yaml).
+The motivation and description of the API can be find in [design doc].
+Definitions of test as well as definitions of individual objects support templating.
+Templates for test definition come with one predefined value - ```{{.Nodes}}```, which represents the number of schedulable nodes in the cluster. \
```

Comment:
nit: please try to split the lines even more so that they fit a single line in this github UI (I think 100 chars)

Reply:
Done.